mv-openapi.yaml

```diff
@@ -0,0 +1,476 @@
+# Warning: this file has been automatically generated. All manual modifications will be lost, see README.md for details.
```

Comment:
If this file is automatically generated it should be removed from the commits?

Reply:
I don't think so.

It can be really usefull to have to main openapi file located somewhere in this repository. For instance if you want put this file on a web server you just have to wget it, not to clone the repo, install npm and run the build.

We just have to don't forget to build it before to push, but this can be tested with a CI tool or git hooks.